### DISCUSSION TITLE
Hyva Compatibility?

### DISCUSSION THREAD

**Discussion by brandontamm** [SnowdogApps/magento2-menu#292] [General]
Support for hyva currently or planned? Any known issues?

**Comment by andrzejborkowski** [comment]
hi @brandontamm,

yes, our dev team works on the compatibility module at the moment (internal ticket DEV-101754). I'll let you know once it will be released.

Regards,
Andrzej

**Reply by CoutoDev** [reply]
Glad to hear that!

**Comment by andrzejborkowski** [comment]
Hi,

just letting you know that our module is already compatible with Hyva theme starting from version 2.24.0

Regards,
Andrzej
